### ISSUE TITLE
Add EventSense

### ISSUE THREAD

**Issue by McTwist** [McTwist/Support_EventScript#16]
Whenever a script is pasted onto the window, store the information, along with comments and whitespace, somewhere so it can be retrieved and added when creating the script.

This might seem most difficult and might result in unexpected output, but if properly done with as little predictions as possible, then this will be a really welcoming addition.

Most certainly, at least make it possible for the editor to keep its content.

**Comment by McTwist** [comment]
In the object, there should be some sort of way to define specific information of how the script is created, making it possible to store exact information about whitespace, newline and comment. It is currently unknown how this is going to be, but most suitable would be to store the info in a field group named `graph`, which will be a list of each line and when an event will be added to it.

The biggest problem would be events that is split up in newlines or got lots of whitespace in them. Also, if the events are changed significantly, how will that modify the existing events? One might be able to achieve that by linking the `graph` to specific lines and somehow get to know the info from how the events are modified.

**Comment by haze** [comment]
I'm just curious, IntelliSense in regards to what information? IntelliSense usually provides in-depth information unlike file-wide autocomplete for named fields and functions, usually global. Does this mean the inclusion of other evented brick names and such?

**Comment by McTwist** [comment]
Yes, it should include lookup for events within the same brick for now, but later on it would support multiple bricks, for instance for labels.
